lib/lightning_web/live/project_live/settings.ex

```diff
@@ -141,16 +141,15 @@ defmodule LightningWeb.ProjectLive.Settings do
     <%= if can_edit_project_user(@current_user, @project_user) do %>
       <.form
         :let={form}
-        for={%{}}
+        for={%{"failure_alert" => @project_user.failure_alert}}
         phx-change="set_failure_alert"
         id={"failure-alert-#{@project_user.id}"}
       >
         <%= hidden_input(form, :project_user_id, value: @project_user.id) %>
         <LightningWeb.Components.Form.select_field
           form={form}
           name="failure_alert"
-          values={[Disabled: "false", Enabled: "true"]}
-          selected={@project_user.failure_alert |> Atom.to_string()}
+          values={[Disabled: false, Enabled: true]}
         />
```

Comment:
@elias-ba I finally think we have the 'correct' way to do this.

`selected` worked in this situation, since it's a default when no value is provided.
Since we're using a form now, we should 'expect' the value to come from the form struct... but since it was an empty map, selected worked just fine.

Since the Form builder functions can deal with booleans, instead of coercing them into strings, if we set the params for the form (just like how a controller gets its params) we can skip setting 'selected'.